examples/describe_repo.d

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env dub
+/+ dub.sdl:
+name "example"
+description "Script that fetched various information about content of the repo"
+authors "Mihails Strasuns"
+copyright "Copyright 2016, Sociomantic Labs"
+license "Boost"
+
+targetType "executable"
+dependency "octod" version="*" path="../"
++/
+
+import octod.core;
+import octod.api.repos;
+
+import std.stdio;
+
+void main ( )
+{
+    Configuration conf;
+    conf.dryRun = true;
+
+    auto client = HTTPConnection.connect(conf);
+    auto repo = client.repository("sociomantic-tsunami/ocean");
+
+    writeln("-----------------------------------------------------------");
+    writeln("List of known git tags belonging to GitHub release:");
+    auto tags = repo.releasedTags();
+    foreach (tag; tags)
+        writefln("%s: %s", tag.name, tag.sha);
+
+    writeln("-----------------------------------------------------------");
+    writeln("Root directory content for the default branch:");
+    writeln();
+    writeln(repo.download("").expectDirectory.listAll());
```

Comment:
Both versions are supported (github will redirect as needed).
Adjusted example to make it obvious.